obspy/taup/tests/test_tau.py

```diff
@@ -107,6 +112,64 @@ def test_p_iasp91_manual(self):
         self.assertAlmostEqual(p_arrival.purist_distance, 35.00, 2)
         self.assertEqual(p_arrival.purist_name, "P")
 
+    @unittest.skipIf(not geobase.HAS_GEOGRAPHICLIB,
+                     'Module geographiclib is not installed')
+    def test_p_iasp91_geo_manual(self):
+        """
+        Manual test for P phase in IASP91 given geographical input.
+
+        This version of the test is used when geographiclib is installed
+        """
+        m = TauPyModel(model="iasp91")
+        arrivals = m.get_travel_times_geo(source_depth_in_km=10.0,
+                                          source_latitude_in_deg=20.0,
+                                          source_longitude_in_deg=33.0,
+                                          station_latitude_in_deg=55.0,
+                                          station_longitude_in_deg=33.0,
+                                          phase_list=["P"])
+        self.assertEqual(len(arrivals), 1)
+        p_arrival = arrivals[0]
+
+        self.assertEqual(p_arrival.name, "P")
+        self.assertAlmostEqual(p_arrival.time, 412.43, 2)
+        self.assertAlmostEqual(p_arrival.ray_param_sec_degree, 8.612, 3)
+        self.assertAlmostEqual(p_arrival.takeoff_angle, 26.74, 2)
+        self.assertAlmostEqual(p_arrival.incident_angle, 26.69, 2)
+        self.assertAlmostEqual(p_arrival.purist_distance, 35.00, 2)
+        self.assertEqual(p_arrival.purist_name, "P")
+
+    def test_p_iasp91_geo_fallback_manual(self):
+        """
+        Manual test for P phase in IASP91 given geographical input.
+
+        This version of the test checks that things still work when
+        geographiclib is not installed.
+        """
+        has_geographiclib_real = geobase.HAS_GEOGRAPHICLIB
+        geobase.HAS_GEOGRAPHICLIB = False
+        taup_geo.HAS_GEOGRAPHICLIB = False
+        tau.HAS_GEOGRAPHICLIB = False
```

Comment:
In principle I suppose only `obspy.geodetics` should care if geographiclib is installed or not. Then this stops being a problem (because `taup.tau` and `taup.taup_geo` won't need to look at `HAS_GEOGRAPHICLIB`).  To do this we need a handful of new features in `obspy.geodetics` - specifically a solution to the direct geodesic problem, and something that looks a bit like geographiclib's "line" object. Something like #1027 is a start in that direction but that does not have everything.